docs-source/content/recent-changes.md

```diff
@@ -10,7 +10,7 @@ This document tracks recent changes to the operator, especially ones that introd
 
 | Date | Version | Introduces backward incompatibilities | Change |
 | --- | --- | --- | --- |
-| April 30, 2019 | v2.2 | no  | Added support for FMW Infrastructure domains. WebLogic server instances are now gracefully shutdown by default and shutdown options are configurable. Operator is now built and runs on JDK 11. 
+| May 6, 2019 | v2.2 | no  | Added support for FMW Infrastructure domains. WebLogic server instances are now gracefully shutdown by default and shutdown options are configurable. Operator is now built and runs on JDK 11. 
```

Comment:
WebLogic server -> WebLogic Server (a legal trademark requirement to always capitalize both WebLogic and Server)